app/code/core/Mage/Customer/Block/Address/Renderer/Default.php

```diff
@@ -131,7 +131,7 @@ public function render(Mage_Customer_Model_Address_Abstract $address, $format =
 
         if ($this->getType()->getHtmlEscape()) {
             foreach ($data as $key => $value) {
-                $data[$key] = $this->escapeHtml($value);
+                $data[$key] = $this->escapeHtml($value ?? '');
```

Comment:
```suggestion
                $data[$key] = $this->escapeHtml($value);
```
Revert after https://github.com/OpenMage/magento-lts/pull/2884